### PR TITLE
Setup tests for relational database schema

### PR DESCRIPTION
**Pull request by r-peschke**
- [x] pytest-fixtures for database session- and single-test-setup
- [x] prepare current database instance in dev-setup with both schemata
- [x] Prevent (some) unneccessary commands like truncate
- [x] Rely environment handling on class Environment
- [x] Start new connection handling with unpooled connections
- [x] Test reset with TRUNCATE TABLE based on triggered creates on tables

### CODE REVIEW

**Review by rrenkert — APPROVED** [review]

**Review by reicda — CHANGES_REQUESTED** [review]
- Autocommit should be set to True as there is a need for transactions containing multiple commands.
- There are some casings that can be improved for a better readability.
- Everything else looks fine to me.

File: openslides_backend/database/db_connection_handling.py
[the remark this answers is not in the document]
Comment (r-peschke) [reply openslides_backend/database/db_connection_handling.py:1]: see conclusion comment from @reicda to stay with autocommit=True

File: tests/datastore/reader/system/history_information/test_history_information.py
[the remark this answers is not in the document]
Comment (r-peschke) [reply tests/datastore/reader/system/history_information/test_history_information.py:1]: done

[the remark this answers is not in the document]
Comment (r-peschke) [reply tests/datastore/reader/system/history_information/test_history_information.py:1]: done

File: tests/datastore/writer/system/test_write_action_worker.py
[the remark this answers is not in the document]
Comment (r-peschke) [reply tests/datastore/writer/system/test_write_action_worker.py:1]: done

File: tests/datastore/writer/system/util.py
[the remark this answers is not in the document]
Comment (r-peschke) [reply tests/datastore/writer/system/util.py:1]: done

File: tests/datastore/writer/system/write/test_create.py
[the remark this answers is not in the document]
Comment (r-peschke) [reply tests/datastore/writer/system/write/test_create.py:1]: done

File: tests/datastore/writer/system/write/test_migration_index.py
[the remark this answers is not in the document]
Comment (r-peschke) [reply tests/datastore/writer/system/write/test_migration_index.py:1]: done

File: tests/datastore/writer/system/write/test_writerequest_data.py
[the remark this answers is not in the document]
Comment (r-peschke) [reply tests/datastore/writer/system/write/test_writerequest_data.py:1]: done

**Review by reicda — APPROVED** [review]

**Pull request merged**